lib/cdo/web_purify.rb

```diff
@@ -10,6 +10,13 @@ module WebPurify
     'ja' => 'jp'
   }.freeze
 
+  WEBPURIFY_URL = URI('http://api1.webpurify.com/services/rest').freeze
+
+  CONNECTION_OPTIONS = {
+    read_timeout: DCDO.get('webpurify_http_read_timeout', 10),
+    open_timeout: DCDO.get('webpurify_tcp_connect_timeout', 5)
```

Comment:
I tried stubbing the `POST` request and using WebMock's `.to_timeout` function to simulate a timeout and test the `read_timeout` and `open_timeout` here, but for some reason, the stub wasn't working. I have a feeling it might have to do with the fact that I was mixing VCR and WebMock in the tests. I'd still love to write that test if anyone has expertise here and would like to pair.